detox/ios/Detox/Actions/NSObject+DetoxActions.m

```diff
@@ -254,6 +309,58 @@ - (void)dtx_pinchWithScale:(CGFloat)scale velocity:(CGFloat)velocity angle:(CGFl
 	_DTXApplyPinch(window, startPoint1, endPoint1, startPoint2, endPoint2, 1.0 / velocity);
 }
 
+- (void)dtx_dragAndDrop:(NSObject *)targetElement targetPositionOffset:(CGPoint)positionOffset initialHoldDuration:(NSTimeInterval)initialHoldDuration speed:(CGFloat)speed
```

Comment:
How come this never gets called? Is it because this is just a draft?

Reply:
Didn't remove it yet, still doing changes. Will be removed next commit